frontend/src/components/Dropdown/SingleSelect/SingleSelectProvider.tsx

```diff
@@ -21,7 +21,7 @@ export interface SingleSelectProviderProps<
 > extends SharedSelectContextReturnProps<Item>,
     FormControlOptions {
   /** Controlled selected value */
-  value: string
+  value: string | undefined
```

Comment:
Any specific reason why this should be switched to undefined? We usually take `''`